README.md

```diff
@@ -1,6 +1,22 @@
 # Orchestra
 
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/unlimitedlabs/orchestra?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+Test status [![Circle CI](https://circleci.com/gh/unlimitedlabs/orchestra.svg?style=svg)](https://circleci.com/gh/unlimitedlabs/orchestra)
+
+Chat with the team [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/unlimitedlabs/orchestra)
+
+Orchestra is an Apache 2-licensed system for coordinating project teams of experts and machines.  Projects can include everything from design teams working on client delivery to newsrooms bringing together reporters, photographers, and editors for a story.  Orchestra supports expert review, so an expert working on any task in a project can get feedback and iteratively refine their work.  It also brings machines and automation onto projects: a crawler can collect content from the web before an expert combs through it, or a classifier can filter out bad data so an analyst spends less time in the noise.
```

Comment:
"design teams working on client delivery" -- the 'client delivery' bit sounds a little stuffy.

Also maybe need some text on extensibility/how to build around Orchestra here: "New workflows can be added with some simple python glue and an html interface."